rosboard/html/js/viewers/meta/Space3DViewer.js

```diff
@@ -90,6 +90,56 @@ class Space3DViewer extends Viewer {
       that.updatePerspective();
     }
 
+    var touchRegion = ZingTouch.Region(this.gl.canvas)
```

Comment:
How is zingtouch installed?  I didn't see any changes in the SRM repo install scripts.

Reply:
Rosboard does not use node, so the library was downloaded and added to rosboard [here](https://github.com/offworld-projects/offworld-rosboard/tree/feature/touch-gestures/rosboard/html/js), under `rosboard/html/js`. This is how the other Javascript libraries used by rosboard are installed. It is called [here](https://github.com/offworld-projects/offworld-rosboard/blob/f1ff28a12c07c14e66adb6c115dc77779c1360ac/rosboard/html/viewer.html#L21) @jwdinius